packages/battery/android/src/main/java/io/flutter/plugins/battery/BatteryPlugin.java

```diff
@@ -30,16 +30,16 @@ public static void registerWith(PluginRegistry.Registrar registrar) {
         new MethodChannel(registrar.messenger(), "plugins.flutter.io/battery");
     final EventChannel eventChannel =
         new EventChannel(registrar.messenger(), "plugins.flutter.io/charging");
-    final BatteryPlugin instance = new BatteryPlugin(registrar.context());
+    final BatteryPlugin instance = new BatteryPlugin(registrar);
```

Comment:
AFAIK, `registrar.context()` constantly returns the same value. If that is correct, this change isn't strictly necessary. Are we doing it anyway for consistency with plugins that need the `registrar` to access the current `Activity`?

Same in other plugins that just need a `Context`, and not an `Activity`.

Reply:
Yes, it's for consistency.